Dockerfile

```diff
@@ -23,14 +22,36 @@ ENV ANDROID_NDK=/opt/ndk/android-ndk-r$NDK_VERSION
 ENV PATH=${PATH}:${ANDROID_NDK}
 
 # install system dependencies
-RUN apt-get update && apt-get install ant autoconf automake curl g++ gcc git libqt5widgets5 lib32z1 lib32stdc++6 make maven npm openjdk-8* python-dev python3-dev qml-module-qtquick-controls qtdeclarative5-dev unzip -y
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        ant \
+        apt-transport-https \
+        autoconf \
+        automake \
+        curl \
+        g++ \
+        gcc \
+        git \
+        gnupg2 \
+        libqt5widgets5 \
+        lib32z1 \
+        lib32stdc++6 \
+        make \
+        maven \
+        python-dev \
+        python3-dev \
+        qml-module-qtquick-controls \
```

Comment:
I am curious is two dependency for, is they redundant ?

Reply:
it was there, and you spotted it because it's more readable now 👍. I'll try to do more cleanups soon.